### PR TITLE
show reward for single councillor on council page

### LINKED ISSUE #4043: [USERSNAP] Council: Councilor Reward is either wrong or unclear
**Taken at**: https://pioneerapp.xyz/#/council
**Fields**:
- Houston, we have a problem.: Council: Councilor Reward is either wrong or unclear
- Comment: The current amount shown can easily be read as 1.16 $JOY / 14,400 blocks but it is really not clear and the maths don&#39;t seem to add up if I use the QN to find this information manually.
- Feedback: tomato#9464

SCREENSHOT
*[Download original image](https://resources.usersnap.com/company/f9f719df-a929-4461-bb6e-46dbfbceffb7/datapoint_screenshot/4f624cd3-1673-40ac-9c6a-c5d2e01ea018-annotated_dbe8836c-a103-4c55-9ab3-826d495d3983.png?etag=ce76c2c0f72626cdc517a04eba2cbcd6)*

**Browser**: Chrome 108 (macOS Catalina)
**Screen size**: 1440x900
**Browser size**: 1396x722
**[Open #368 in Usersnap](https://app.usersnap.com/l/feedback/83f33c52-3a80-42c1-8da8-5810864cb705?utm_medium=integration&utm_campaign=integration&utm_content=open)**

Powered by **[Usersnap](https://usersnap.com/?utm_source=product&amp;utm_medium=poweredbylink&amp;utm_campaign=github_entry)**.

### PR DESCRIPTION
**Pull request by kdembler**
Let's show reward per week but for a single council members, not for the full council, that's very confusing.

Close #4043

### CODE REVIEW

File: packages/ui/src/app/pages/Council/Council.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ export const Council = () => {
 
   const isCouncilorLoading = !isRefetched && (isLoading || isLoadingCouncilors)
 
-  const rewardPerDay = useMemo(() => reward?.period?.mul(reward?.amount ?? asBN(0)) ?? asBN(0), [reward])
+  const rewardPerDay = useMemo(() => reward?.period?.mul(reward?.singleCouncilorAmount ?? asBN(0)) ?? asBN(0), [reward])
   const main = (
     <MainPanel>
       <StatisticsStyle>
```
Comment (thesan) [inline packages/ui/src/app/pages/Council/Council.tsx:47]: To make things 100% clear could you rename "Councilor Reward" to "Per Councilor Reward" please

**Review by thesan — CHANGES_REQUESTED** [review]

File: packages/ui/src/app/pages/Council/Council.tsx (hunk 1 above)
Comment (kdembler) [reply packages/ui/src/app/pages/Council/Council.tsx:47]: You mean in the UI here?

Comment (thesan) [reply packages/ui/src/app/pages/Council/Council.tsx:47]: Yes

Comment (kdembler) [reply packages/ui/src/app/pages/Council/Council.tsx:47]: Changed to `Single Councillor Reward`

**Review by thesan — APPROVED** [review]
Perfect! Thanks!

**Pull request merged**